app/models/orphan.rb

```diff
@@ -1,14 +1,35 @@
 class Orphan < ActiveRecord::Base
 
-  NEW_SPONSORSHIP_SORT_SQL = '"orphan_sponsorship_statuses"."name", "orphans"."priority"  ASC'
+  enum status: [
+    :active,
+    :inactive,
+    :on_hold,
+    :under_revision
+  ]
+
+  enum sponsorship_status: [
+    :unsponsored,
+    :sponsored,
+    :previously_sponsored,
+    :sponsorship_on_hold
+  ]
+
   AGE_OF_ELIGIBILITY_TO_JOIN = 22
   VALID_GESTATION = 1
 
+  SPONSORSHIP_STATUS_ORDERING = %Q{
+  CASE WHEN sponsorship_status = '#{Orphan.sponsorship_statuses[:previously_sponsored]}' THEN '1'
+       WHEN sponsorship_status = '#{Orphan.sponsorship_statuses[:unsponsored]}'          THEN '2'
+  END ASC
+  }
```

Comment:
Client requested ordering of eligible orphans as 'Previously Sponsored', 'Unsponsored'. Prior implementation relied on the fact that the former precedes the latter alphabetically. By contrast, this SQL explicitly specifies the order.